sbin/upload-salt-reports

```diff
@@ -108,6 +108,17 @@ def upload(jobs):
         if job['job']['result'] == {}:
             continue
 
+        # Convert duration to seconds to conform with Puppet run
+        # duration, and in turn, the run time on Foreman reports
+
+        for node, states in job['job']['result'].iteritems():
+            if isinstance(states, dict):
+                for state, params in states.iteritems():
+                    for key, value in params.iteritems():
```

Comment:
Is params guaranteed to be a dict here or could we end up with some error if salt gives us bad input?